### DISCUSSION TITLE
Can not change Image or Label with code

### DISCUSSION THREAD

**Discussion by rulai-yinqiang** [harborsiem/WindowsRibbon#19] [Q&A] [answered]
The code not work
>     Public Sub New()
>         InitializeComponent()
>         Rib_CmdPaste = New RibbonButton(Ribbon1, CUInt(RibbonMarkupCommands.Rib_CmdPaste))
>         Rib_CmdPaste.SmallImage = Ribbon1.ConvertToUIImage(My.Resources.Paste_16)
>         Rib_CmdPaste.Label = "New Lable"
>     End Sub

**Comment by harborsiem** [comment, marked as answer]
You have to set SmallImage and Label in the Load event of the Form or later on. Then your code will work.

The Ribbon Framework and also the Image converter have to be initialized first before one can set these values. This is done before the Load event fires.
